### DISCUSSION TITLE
Has anyone tried running khoj on a AWS server?

### DISCUSSION THREAD

**Discussion by fkgruber** [khoj-ai/khoj#452] [Q&A]
I wonder what is the cost of running khoj on the minimum viable server in AWS or some other commercial service.

**Comment by sabaimran** [comment]
Hey @fkgruber! We've tried this out. I think it should run you around ~35 USD/mo if you run with `t2.medium` in EC2. I can put together some instructions for this if there's enough interest.

**Comment by debanjum** [comment]
Converting this into a Github discussion for convenience

**Comment by fkgruber** [comment]
Thanks! and just to be clear there is no way Khoj can run on the Free tier AWS is that right?
